### DISCUSSION TITLE
Crop destruction possible or not ?

### DISCUSSION THREAD

**Discussion by bojanh66** [Courseplay/Courseplay_FS22#2030] [Q&A] [answered]
Is it possible to add a code line in Courseplay to enable crop destruction, mainly for personal use?
I am not a programmer by all means, so i dont know if this is possible or not?
The main reason is mostly visual tramlines when spraying wheat, barley etc, i would like to see Courseplay worker create tramlines.

**Comment by Tensuko** [comment, marked as answer]
That is not possible as Giants doesn't have crop destruction for AI.
